samples/sensor/isl29125/Kconfig

```diff
@@ -0,0 +1,8 @@
+mainmenu "Application"
```

Comment:
This file doesn't seem necessary, since you don't have any application-specific configs. I think you can remove it.